addons/api/mirage/factories/host-catalog.js

```diff
@@ -70,6 +70,16 @@ export default factory.extend({
           client_id: faker.string.uuid(),
           subscription_id: faker.string.uuid(),
         };
+      case 'gcp':
+        return {
+          disable_credential_rotation: faker.datatype.boolean(),
+          project_id: faker.string.uuid(),
+          client_email: faker.internet.email(),
+          target_service_account_id: faker.string.uuid(),
+          zone: `us-${faker.location.cardinalDirection()}-${faker.number.int(
+            9,
+          )}`,
```

Comment:
```suggestion
          zone: `us-${faker.location.cardinalDirection()}${faker.number.int(
            9,
          )}-${faker.string.fromCharacters('abc')}`,
```
This is more accurate to what gcp zones looks like it seems

Reply:
ah yes that looks right, thanks for the suggestion!